src/main/java/au/org/emii/ncdfgenerator/CreateWritable.java

```diff
@@ -2,29 +2,44 @@
 package au.org.emii.ncdfgenerator;
```

Comment:
Can we standardise on the package and project name (i.e. to be the same)?

Also, `netcdf` seems to be the more common abbreviation (?) (at least when I looked at [thredds source code](https://github.com/Unidata/thredds/search?l=java&q=netcdf&type=Code&utf8=%E2%9C%93) for example), would that be better?

e.g. `au.org.emii.netcdfgenerator` / netcdf-generator respectively